ui/src/message_popup/playtest/smithTurner_scenarios.jsx

```diff
@@ -0,0 +1,373 @@
+/* 
+Scenarios page for the Mr. Smith Microaggression simulation
+Code adapted from the Turner_scenarios.js file
+MIT Teaching Systems Lab
+ */
+import React from 'react';
+import hash from '../../helpers/hash.js';
+
+export type ResponseT = {
+  choice:string,
+  question:QuestionT,
+  downloadUrl:string
+};
+
+export type QuestionT = {
+  id:number,
+  imgId:string,
+  youTubeId:string,
+  start: number,
+  end: number,
+  type:string, // A string that gets displayed as the page heading
+  stage:string // one of 'info', 'prereflect', 'scenario', 'postreflect'
+};
+
+
+// Make questions and choices.
+export const SmithScenarios = {
+  choices() {
+    return [];
+  },
+
+  getSlides() {
+    const slides:[QuestionT] = [];
+
+
+// -----------------------------------------------------
+// OVERVIEW
+// -----------------------------------------------------
+
+    slides.push({type: 'Overview', stage: 'info', text: 
+      
+    `1. Set Context
+    Imagine yourself as a teacher situated in the context of the particular school, classroom and subject.
+    
+    2. Background
+    A little bit of background information on the lesson being taught.
+    
+    3. Try it!
+    When you're ready, you'll go through a set of short scenes that simulate moments in the classroom. Note what you observe.
+    
+    4. Lenses
+    Try another set of scenes, this time from a new perspective.`
+  });
+
+  // -----------------------------------------------------
+  //Context
+  // -----------------------------------------------------
+    slides.push({type: 'Context', stage: 'info', text: 
+      `Imagine you're a teacher at a suburban high school. Your friend, and colleague Mr. Smith asks you to sit in and observe his CS class for a day.
+
+The class, taught by Mr. Smith, has 6 students.The demographics of the class are as follows:
+
+5 Male Students
+1 Female Students
+4 White Students (3 male, 1 female)
+2 African American Students (2 male)`
+  });
+
+
+
+  // -----------------------------------------------------
+  // BACKGROUND
+  // -----------------------------------------------------
+
+    slides.push({type: 'Background', stage: 'info', text: 
+      `Mr. Smith will be teaching a class on encoding and sending formatted text. 
+
+In doing so his students will be working with the standard internet encoding protocol, ASCII. The class will be using pair programming to complete a project developing their own protocol for sending formatted (i.e bold, italicized, etc) text. 
+
+`
+    });
+
+    slides.push({type: 'Background', stage: 'info', text: 
+      `Useful Knowledge:
+
+ASCII, is a protocol for representing alphabetic, numeric, and special characters with specific 7-bit binary numbers. 
+
+Binary is a number scheme that represents data through the use of only two possible digits: 0 or 1.
+
+The term “Internet Simulator” will come up in some scenes. It is a tool which the students will use to test communication protocols on the internet. 
+
+-- -- -- -- --
```

Comment:
What's with this, is it a divider?  You can use `<Divider>` if you want a visual divider line, or you could put this on another slide.